### DISCUSSION TITLE
Q: How to bookmark a channel without seeing the videos?

### DISCUSSION THREAD

**Discussion by furrnace** [FreeTubeApp/FreeTube#2544] [Q&A]
I started using freetube for macos without linking it to an account on youtube.
I see how I can subscribe to a channel, which gives me all the videos of that channel in the subscriptions tab. I don't want that.
I just want to add a channel and then click on that in the channels tab.
I haven't yet understood how to do that, I don't see a "add channel" or "join channel" button anywhere.
how can I do this?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Unfortunately it is not possible to only bookmark a channel without seeing the videos appear in the a Subscription tab.
